docs/docs/guide/contentful.mdx

```diff
@@ -0,0 +1,140 @@
+
+![Contentful entry with GrowthBook experiment variations](/images/guides/contentful-experiment.png)
+
+### Experiment
+
+Once configured, you're ready to start your experiment. This syncs the data created in Contentful with GrowthBook, so you can analyze the experiment results and see which version of your test performs best.
+
+The diagram below illustrates how the above components work together.
+
+![Contentful and GrowthBook flow diagram](/images/guides/contentful-gb-diagram.png)
+
+Continue reading to learn how to set up GrowthBook and Contentful to run A/B tests on your site.
+
+## Set up GrowthBook
+
+### 1. Create an SDK Connection
+
+An SDK connection is a link between your GrowthBook account, your frontend codebase, and the Contentful plugin. It allows GrowthBook to serve the correct content variations to your users.
+
+1. Log in to GrowthBook
+2. Navigate to **SDK Configuration** → **SDK Connections** in the sidebar
+3. Click **Add SDK Connection**
+4. Select **JavaScript** or **React** as your language
+5. Save your **API Host** URL for later use
+
+![GrowthBook SDK connection](/images/guides/contentful-sdk-connection-data.png)
+
+### 2. Configure data source
+
+A data source is the connection between GrowthBook and your analytics platform. This connection allows GrowthBook to analyze your experiment data and provide insights. It's also informs Contentful which data source to use when creating experiments.
+
+1. Go to **Metrics and Data** → **Data Sources** in GrowthBook
+2. Click **Add Data Source**
+3. Choose your preferred analytics platform
+4. Follow the connection guide
+5. Save your data source ID (found in the URL of your data source page)
+
+![GrowthBook data source ID location](/images/guides/contentful-data-source-id.png)
+
+### 3. Generate API key
+
+To connect Contentful to GrowthBook, you need an API key with admin permissions. This key allows Contentful to communicate with GrowthBook and sync your experiments.
+
+1. Navigate to **Settings** → **API Keys**
+2. Click **Create New Key**
+3. Select `admin` role
+4. Save the generated key securely
+
+![GrowthBook API test](/images/guides/contentful-api-keys.png)
+
+## Install and configure the GrowthBook app
+
+1. Go to **Apps** → **Marketplace**
+2. Search for GrowthBook
+3. Click **Install**
+4. Configure the plugin:
+
+- Enter your GrowthBook API host URL from step 1
+- Enter your data source ID from step 2
+- Enter your Admin API key from step 3
+- Click **Save**
+
+## Run your first experiment
+
+With everything configured, you're ready to run your first experiment.
+
+1. Add the **GrowthBook Experiment** to a page
+   In Contentful, open the content entry where you want to add the experiment. Add the **GrowthBook Experiment** content type. (You may have to update the page to accept the new content type.)
```

Comment:
I updated this to be more explicit; however, I'm understanding our audience to be somewhat familiar already with Contentful, so that they'll already have a good understanding of my content models/entries work.